package.json

```diff
@@ -4,7 +4,18 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "hardhat compile --config ./hardhat.config.js",
+    "prebuild": "rimraf ./artifacts/ ./cache/",
+    "test": "hardhat test --config ./hardhat.config.js",
+    "profile": "REPORT_GAS=true hardhat test --config ./hardhat.config.js",
+    "lint:check:format": "prettier --check **.{sol,js}",
+    "lint:check:solhint": "solhint contracts/**/*.sol",
+    "lint:check:eslint": "eslint . --ext js,ts",
+    "check": "yarn lint:check:format && yarn lint:check:solhint && yarn lint:check:eslint",
+    "lint:fix:format": "prettier --write **.{sol,js}",
+    "lint:fix:eslint": "eslint --fix . --ext js",
+    "fix": "yarn lint:fix:format && yarn lint:fix:eslint",
+    "verify": "npx hardhat verify"
```

Comment:
verify needs arguments. This is better called from the terminal using npx commands. We can remove it from here